ui/app/styles/components/global-flash.scss

```diff
@@ -11,14 +11,4 @@
   position: fixed;
   width: 95%;
   z-index: 300;
-
```

Comment:
i appreciate that you're also removing the css associated with the old alert banners!